src/sentry/api/endpoints/issues_resolved_in_release.py

```diff
@@ -0,0 +1,54 @@
+from __future__ import absolute_import
+
+from rest_framework.response import Response
+
+from sentry.api.base import DocSection
+from sentry.api.bases.project import ProjectEndpoint, ProjectPermission
+from sentry.api.exceptions import ResourceDoesNotExist
+from sentry.api.serializers import serialize
+from sentry.api.serializers.models.group import StreamGroupSerializer
+from sentry.models import (
+    Group, GroupCommitResolution,
+    Release, ReleaseCommit,
+)
+
+
+class IssuesResolvedInReleaseEndpoint(ProjectEndpoint):
+    doc_section = DocSection.RELEASES
+    permission_classes = (ProjectPermission,)
+
+    def get(self, request, project, version):
+        """
+        List issues to be resolved in a particular release
+        ````````````````````````
+
+        Retrieve a list of issues to be resolved in a given release.
+
+        :pparam string organization_slug: the slug of the organization the
+                                          release belongs to.
+        :pparam string project_slug: the slug of the project associated with the release.
+        :pparam string version: the version identifier of the release.
+        :auth: required
+        """
+        try:
+            release = Release.objects.get(version=version, organization=project.organization)
+        except Release.DoesNotExist:
+            raise ResourceDoesNotExist
+
+        commits = ReleaseCommit.objects.filter(
+            release=release,
+        ).select_related('commit')
+
+        commitresolutions = GroupCommitResolution.objects.filter(
+            commit_id__in=[rc.commit.id for rc in commits]
+        )
+        groups = Group.objects.filter(project=project,
+            id__in=[cr.group_id for cr in commitresolutions]
+        )
```

Comment:
Similar to the other suggestion, can't this be rewritten as:

```python
groups = Group.objects.filter(
    project=project,
    id__in=GroupCommitResolution.objects.filter(
        commit_id__in=ReleaseCommit.objects.filter(
            release=release,
        ).values_list('commit_id', flat=True),
    ).values_list('group_id', flat=True),
)
```

To yield the same results with 1 query instead of 3 here?